bitarray.go

```diff
@@ -265,3 +265,23 @@ func (bits *BitArray) ToArray() []int {
 
 	return ints
 }
+
+func (bits *BitArray) ToByteArray() []byte {
+	return bits.bytes
```

Comment:
I think `Bytes()` is more precise since there is no copy or conversion, and we need some doc comment to clarify that the changes to the bytes will affect the result of future operations from BitArray.